Algorithms/Encoders/NYSIISEncoder.cs

```diff
@@ -0,0 +1,197 @@
+using System;
+using System.Linq;
+using System.Collections.Generic;
+
+namespace Algorithms.Encoders
+{
+    /// <summary>
+    /// Class for NYSIIS encoding strings.
+    /// </summary>
+    public class NYSIISEncoder
+    {
+        private static char[] Vowels = new char[] { 'A', 'E', 'I', 'O', 'U' };
+
+        /// <summary>
+        /// Creates a new instance of the NYSIISEncoder.
+        /// </summary>
+        public NYSIISEncoder() {}
```

Comment:
Codacy found an issue: [Remove this redundant constructor.](https://app.codacy.com/manual/siriak/C-Sharp/pullRequest?prid=6311443)